### PR TITLE
modify branch

### PR DESCRIPTION
**Pull request by sugo-shota**
add manual & images

### CODE REVIEW

**Comment by youyo** [comment]
Please merge the latest code before your pull request.

**Comment by sugo-shota** [comment]
thx, I merged the latest.

File: RaspberryPi/iot-bcore.py [hunk 1]
```diff
@@ -4,7 +4,6 @@
 from AWSIoTPythonSDK.MQTTLib import AWSIoTMQTTClient
 import os
 
-
```
Comment (youyo) [inline RaspberryPi/iot-bcore.py:7]: expected 2 blank lines

File: RaspberryPi/iot-bcore.py [hunk 2]
```diff
@@ -76,8 +75,7 @@ def subscribe(self):
         while True:
             self.myAWSIoTMQTTClient.subscribe(self.topic, 1, self.customCallback)
 
-
```
Comment (youyo) [inline RaspberryPi/iot-bcore.py:79]: expected 2 blank lines after class or function definition by pep8

File: RaspberryPi/iot-bcore.py [hunk 3]
```diff
@@ -76,8 +75,7 @@ def subscribe(self):
...
 if __name__ == '__main__':
     myAWSIoTMQTTClient = AwsIotClient()
     myAWSIoTMQTTClient.connect()
-    myAWSIoTMQTTClient.subscribe()
```
Comment (youyo) [inline RaspberryPi/iot-bcore.py:83]: no newline at end of file

**Comment by youyo** [comment]
I write some comments.
I would like you to follow pep8 and flake8.

**Pull request closed without merging**